moveit_ros/moveit_servo/src/utilities.cpp

```diff
@@ -179,4 +179,82 @@ bool applyJointUpdate(rclcpp::Clock& clock, const double publish_period, const E
   return true;
 }
 
+void commandToMoveGroupFrame(geometry_msgs::msg::TwistStamped& cmd, const servo::Params& servo_params,
+                             const Eigen::Isometry3d& tf_moveit_to_robot_cmd_frame,
+                             const Eigen::Isometry3d& tf_moveit_to_ee_frame,
+                             const moveit::core::RobotStatePtr& current_state)
+{
+  Eigen::Vector3d translation_vector(cmd.twist.linear.x, cmd.twist.linear.y, cmd.twist.linear.z);
+  Eigen::Vector3d angular_vector(cmd.twist.angular.x, cmd.twist.angular.y, cmd.twist.angular.z);
+
+  // If the incoming frame is empty or is the command frame, we use the previously calculated tf
+  if (cmd.header.frame_id.empty() || cmd.header.frame_id == servo_params.robot_link_command_frame)
```

Comment:
Shouldn't the case that the cmd's `frame_id` field be an error case or at least print a warning, that servo is making assumptions here?

Reply:
Yes that is a good idea.
Should print a warning here.